Chp9Lsn11/OptionTester/UnitTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Chp9Lsn11;
+using UserInputHandler;
+
+namespace OptionTester
+{
+    [TestClass]
+    public class UnitTest
+    {
+        [TestMethod]
+        public void TextMenu_Test()
+        {
+            string[] possibleInput = { "reverse", "average", "equation" };
+            int i = 0;
+
+            foreach (string command in possibleInput)
+            {
+                string userCommand = Program.TextMenu(possibleInput[i]);
```

Comment:
Revise the code, as needed, so that you can test a bad string like "aardvark" and get some kind of error back.  This would allow you to "assert" that an error message was received as expected.